packages/bot-web-ui/src/app/app.jsx

```diff
@@ -2,19 +2,10 @@ import '../public-path'; // Leave this here (at the top)! OK boss!
 import React from 'react'; // eslint-disable-line import/first
 import { Loading } from '@deriv/components';
 import { DBot, ServerTime, ApiHelpers } from '@deriv/bot-skeleton'; // eslint-disable-line import/first
-import {
-    Audio,
-    BlocklyLoading,
-    BotFooterExtensions,
-    BotNotificationMessages,
-    MainContent,
-    QuickStrategy,
-    RunPanel,
-    RoutePromptDialog,
-    Toolbar,
-    NetworkToastPopup,
-} from 'Components';
-import { LocalStore } from '@deriv/shared';
+import { BotNotificationMessages, NetworkToastPopup, QuickStrategy } from 'Components';
+import Audio from '../components/audio';
+import RoutePromptDialogue from '../components/route-prompt-dialog';
+import BotFooterExtensions from '../components/bot-footer-extensions';
```

Comment:
Why are we importing these separately? can be imported from Components yeah?

Reply:
Yes, we can use components. Removed and used a common import.